sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-CSGD.e2e-spec.ts

```diff
@@ -0,0 +1,92 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+import { YesNoOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-CSGD.`, () => {
+  it(
+    "Should determine CSGD as eligible when total assessed need is greater than 0 " +
+      ", total eligible dependant is at least 1 and total family income is less than the " +
+      "threshold for a married student.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+      assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
+        YesNoOptions.Yes;
+      assessmentConsolidatedData.studentDataDependantstatus = "independant";
+      assessmentConsolidatedData.studentDataDependants = [
```

Comment:
We have factories like `createFakeStudentDependentEligible` to generate dependents.
Can you please take a look at the below?
https://github.com/bcgov/SIMS/blob/3b85fd10ea4bafd971a1ed9725219dccb2d64076/sources/packages/backend/workflow/test/2023-2024/fulltime-assessment/eligibility/fulltime-assessment-2023-2024-eligibility-CSGD.e2e-spec.ts#L19-L44